README.md

```diff
@@ -32,14 +32,16 @@ This example demonstrates how to open a WebSocket URL:
 
 ```python
 import trio
+import logging
 from trio_websocket import open_websocket_url
 
 
 async def main():
     try:
-        async with open_websocket_url('wss://localhost/foo') as ws:
+        async with open_websocket_url('ws://127.0.0.1:8000') as ws:
```

Comment:
This endpoint is just as likely not to work for anyone running the demo.  I'd prefer using a dedicated example domain.  Use of "wss" and path emphasize SSL and path support of the library.
```
open_websocket_url('wss://example.com/ws_path')
```

Reply:
yea that seems to work well :)